Resources/doc/reference/annotations.rst

```diff
@@ -38,7 +38,7 @@ If you need to disable this for some reason, you can do this via the configurati
 Define Admins
 ^^^^^^^^^^^^^
 
-All you have to do is include Sonata\AdminBundleAnnotations and define the values you need.
+All you have to do is include `Sonata\AdminBundle\Annotations` and define the values you need.
```

Comment:
Please use double backticks, and I don't think there is supposed to be an s at the end, is there?